dotnet/src/Skills/Skills.OpenAPI/Extensions/KernelOpenApiExtensions.cs

```diff
@@ -252,7 +252,10 @@ async Task<SKContext> ExecuteAsync(SKContext context)
         {
             try
             {
-                var runner = new RestApiOperationRunner(new HttpClient(), authCallback);
+                // User Agent may be a required request header fields for some Rest APIs,
+                // but this detail isn't specified in OpenAPI specs, so including for all Rest APIs imported
+                var userAgent = "Microsoft-Semantic-Kernel";
```

Comment:
Would be helpful for other applications to be able to override this value? Consider making this a parameter with a default value.

Reply:
Done